Sources/ComposableArchitecture/Reducer.swift

```diff
@@ -339,4 +464,12 @@ public struct Reducer<State, Action, Environment> {
   ) -> Effect<Action> {
     self.reducer(&state, action, environment)
   }
+
+  func receive(on scheduler: SchedulerType) -> Self {
```

Comment:
Can this be done in an extension in a separate class to provide simpler diffs in the future?
Also some docs would be nice at least for what problem this is solving.

Reply:
I get the other changes which require private vars cannot b.e

Reply:
Good note; Moved this implementation into `Store+Scheduling`.